cypress/e2e/course_board/createEditDeleteColumnInBoard.feature

```diff
@@ -43,7 +43,7 @@ Feature: Course Board - To create, edit and delete column in the course board
         Then I see in dialog box option for multi-column board
         Then I see in dialog box option for single column board
         When I choose multi-column board in the dialog box
-        Then I see the page Course Board detail
+        Then I see the page Course Board details
```

Comment:
I think we have to adapt the same changes as we did in single column board feature. Also, it would be better for readability and understanding to change the feature file name from `createEditDeleteColumnInBoard` to `createEditDeleteMultiColumnBoard`.